src/main/java/org/xbill/DNS/ResolverConfig.java

```diff
@@ -53,9 +56,7 @@
 		String OS = System.getProperty("os.name");
 		String vendor = System.getProperty("java.vendor");
 		if (OS.contains("Windows")) {
-			if (OS.contains("95") ||
-				OS.contains("98") ||
-				OS.contains("ME"))
+			if (Stream.of("95", "98", "ME").anyMatch(OS::contains))
```

Comment:
Do we still want to support these ancient Windows versions?

Reply:
No